admin/src/pages/UserSearch.spec.js

```diff
@@ -82,8 +82,8 @@ describe('UserSearch', () => {
             searchText: '',
             currentPage: 1,
             pageSize: 25,
-            notActivated: null,
-            isDeleted: null,
+            filterByActivated: null,
+            filterByDeleted: null,
```

Comment:
Why is the default value null and not false?
```suggestion
            filterByActivated: false,
            filterByDeleted: false,
```

Reply:
Because we switsch between `null` and `false`.
And we switsch between `null` and `true` in the other case.